<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1647</width>
    <height>1032</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9" rowstretch="2,5" columnstretch="1,6">
    <item row="0" column="0" rowspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>起始节点列表</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_26">
            <property name="text">
             <string>F</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QCheckBox" name="checkBox_24">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_27">
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QCheckBox" name="checkBox_30">
            <property name="text">
             <string>J</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_21">
            <property name="text">
             <string>A</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_32">
            <property name="text">
             <string>L</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="checkBox_34">
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBox_33">
            <property name="text">
             <string>M</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_31">
            <property name="text">
             <string>K</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QCheckBox" name="checkBox_35">
            <property name="text">
             <string>O</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_22">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QCheckBox" name="checkBox_25">
            <property name="text">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QCheckBox" name="checkBox_29">
            <property name="text">
             <string>I</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox_28">
            <property name="text">
             <string>H</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkBox_23">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_36">
            <property name="text">
             <string>P</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBox_37">
            <property name="text">
             <string>Q</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="checkBox_38">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QCheckBox" name="checkBox_39">
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QCheckBox" name="checkBox_40">
            <property name="text">
             <string>T</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>目的节点列表</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>C</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>B</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkBox_9">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_6">
            <property name="text">
             <string>F</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_8">
            <property name="text">
             <string>H</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox_12">
            <property name="text">
             <string>I</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QCheckBox" name="checkBox_11">
            <property name="text">
             <string>J</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_10">
            <property name="text">
             <string>K</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_7">
            <property name="text">
             <string>L</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_15">
            <property name="text">
             <string>M</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBox_16">
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="checkBox_14">
            <property name="text">
             <string>O</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_13">
            <property name="text">
             <string>P</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_19">
            <property name="text">
             <string>Q</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBox_20">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="checkBox_18">
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QCheckBox" name="checkBox_17">
            <property name="text">
             <string>T</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>发送策略</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>停止发送</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>发送消息</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_5">
              <property name="text">
               <string>单次发送</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_7">
              <property name="text">
               <string>多次发送</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>发送次数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_times">
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButton_6">
              <property name="text">
               <string>循环发送</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>发送间隔(毫秒)：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_interval">
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>建立连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>命令记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView">
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>100</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>发送记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_2">
             <column>
              <property name="text">
               <string>编号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>节点名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>发送模式</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>发送序号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>发送时间</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>发送内容</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>接收记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>编号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>节点名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>接收序号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>发送时间</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>接收时间</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>时间间隔</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>接收内容</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1647</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
